src/ln/channel.rs

```diff
@@ -358,12 +379,24 @@ impl Channel {
 		1000 // TODO
 	}
 
+	fn derive_minimum_depth(channel_value_satoshis_msat: u64, push_msat: u64) -> u32 {
```

Comment:
After this patch CONF_TARGET is used in some places, derive_minimum_depth in others, which is wrong. Best to probably just remove CONF_TARGET and use a constant in the function so that its not easy to get it wrong.